src/module/actor/actor.js

```diff
@@ -575,11 +576,15 @@ export class ActorSFRPG extends Actor {
         });
     }
 
-    async rollSkillCheck(skillId, skill, options = {}) {
+    async rollSkillCheck(skillId, skill, options = {}, modified = null) {
         let parts = [];
         let data = this.getRollData();
 
         parts.push(`@skills.${skillId}.mod`);
+
+        if (modified != null && modified != undefined) {
```

Comment:
Don't know enough about JS patterns to know if the undefined check is necessary.

Reply:
General JS shenanigan is that you should use === and !== instead of != and ==. You don't have to test for both null and undefined, in this case I would just put if (!modified) instead.